optuna/__init__.py

```diff
@@ -2,9 +2,8 @@
 from optuna import exceptions
 from optuna import integration
 from optuna import logging
-from optuna import multi_objective
-from optuna import pruners
 from optuna import samplers
+from optuna import pruners
```

Comment:
`multi_objective` was importing `BaseSampler` inside, so `samplers` was imported when importing `multi_objective`.
As `pruners` requires `samplers` before, we need to change the import order; otherwise, we will yield an import error.